webapp/app.py

```diff
@@ -718,6 +735,7 @@ def __init__(self):
         handlers = [
             (r"/api/v1/monitoring/health/?", HealthHandler),
             (r"/api/v1/apispec/?", ApiSpecHandler),
+            (r"/api/v1/version/?", VersionHandler),
```

Comment:
should this really be in `/api/v1` endpoint? maybe just `/version` ?

Reply:
I believe `/api/version` should return 1 for now. This looks like version _of_ api. Maybe in this case will use `/app/version` ?

Reply:
Ok, it's not so critical. `/api/v1/version/` is good as well.